packages/dd-trace/src/appsec/sdk/set_user.js

```diff
@@ -22,6 +24,17 @@ function setUser (tracer, user) {
   }
 
   setUserTags(user, rootSpan)
+  rootSpan.setTag('_dd.appsec.user.collection_mode', 'sdk')
```

Comment:
should we have a constant for `_dd.appsec.user.collection_mode`? it is used in several places

Reply:
We have a lot of hardcoded tags everywhere in the codebase, and we don't have any enum file for appsec yet. I would definitely like to do that, but not sure it makes sense in this PR ? I'd rather make a subsequent PR where I remove all hardcoded tags from our codebase, and put them into an enum file. WDYT ?